index.html

```diff
@@ -518,6 +511,45 @@ <h4>I still have questions. Who can I ask?</h4>
         </div>
       </section>
 
+      <section id="links" class="wrapper style3 fade-up">
+        <div class="inner">
+          <h2>Links and Resources</h2>
+          <div class="features">
+            <section>
+              <span class="icon solid major fa-code"></span>
+              <h3>Code of Conduct</h3>
+              <p>All participants, humans and bots must abide by this <a
+                href="https://docs.google.com/document/d/e/2PACX-1vRGokbfY-E2Dh8CK1HoQMxQbvmyj-9DW9O1vLqXSP3NySQY8_sjvRAJmhFzAkgRhT9vGVtb8Kpx20RI/pub"
+                >Code of Conduct.</a
+              ></p>
+            </section>
+            <section>
+              <span class="icon solid major fab fa-slack"></span>
+              <h3>Join the Slack</h3>
+              <p>Start talking to people by <a
+                href="https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ"
+                >joining our Slack.</a
+              ></p>
+            </section>
+            <section>
+              <span class="icon solid major fa-link"></span>
+              <h3>Sign in to the Slack</h3>
+              <p>For people who have already created a slack account <a href="https://codevid-19.slack.com">
```

Comment:
Capitalize "slack"

Reply:
oops yup

Reply:
actually I am just deleting this because it is the same link as join the slack and you said we could remove it in the clubhouse ticket about slack sidebars